qa/tasks/radosgw_admin_rest.py

```diff
@@ -464,6 +464,11 @@ def task(ctx, config):
     assert out['usage']['rgw.main']['num_objects'] == 1
     assert out['usage']['rgw.main']['size_kb'] > 0
 
+    # TESTCASE 'bucket-stats6', 'bucket', 'stats', 'non-existent bucket', 'fails, 'bucket not found error'
+    (ret, out) = rgwadmin_rest(admin_conn, ['bucket', 'info'], {'bucket' : 'doesnotexist'})
+    assert ret == 404
+    assert out['Code'] == 'NoSuchBucket'
```

Comment:
this test case isn't passing - we get NoSuchKey here

https://tracker.ceph.com/issues/48073

Reply:
note that this runs under the rgw/singleton suite, which wasn't covered in https://pulpito.ceph.com/mkogan-2020-10-15_11:08:15-rgw:verify-rgw-issue-45193_i04-distro-basic-smithi/. please schedule full rgw suites instead of just rgw/verify

Reply:
> @nickjanus @mkogan1 I don't want @cbodley's comment to get lost up there. Please always run the full rgw suite and not just rgw/verify. My QA run got snagged here and I'm having to revert, rebuild, and retest as a result.

@ivancich sincerely apologizing for that, noted for future.

Reply:
@cbodley sorry, will re-submit full suite run for this PR + #37929